gdcdictionary/schemas/treatment.yaml

```diff
@@ -100,12 +145,36 @@ properties:
         $ref:
           - "_terms_enum.yaml#/not_reported/common"
 
+  lesions_treated_number:
+    $ref:
+      - "_terms.yaml#/lesions_treated_number/common"
+    type: number
+
   number_of_cycles:
     $ref:
       - "_terms.yaml#/number_of_cycles/common"
     type: integer
     minimum: 0
 
+  number_of_fractions:
+    $ref:
+      - "_terms.yaml#/number_of_fractions/common"
+    type: number
+
+  prescribed_dose:
+    $ref:
+      - "_terms.yaml#/prescribed_dose/common"
+    type: number
```

Comment:
new prop, so maybe safe to add min of 0? (or whatever appropriate min).

Reply:
Added here too.

Reply:
looks good